package.json

```diff
@@ -68,15 +68,15 @@
     "classnames": "^2.3.2",
     "credit-card-type": "^8.0.0",
     "dompurify": "^2.4.0",
-    "downshift": "^3.2.10",
+    "downshift": "^7.6.0",
```

Comment:
`downshift` did not introduce any breaking changes from version `4` to `7`,

Major changes according to ChatGPT:

> Version 4.0.0: This version, released in January 2020, added support for the React Hooks API, which allows components to manage state and lifecycle methods without using classes. It also introduced new features like the useSelect hook, which simplifies the creation of select components, and improvements to accessibility and TypeScript support.
>
> Version 5.0.0: This version, released in May 2020, added support for multiple item selections and improved the accessibility and TypeScript support of the library. It also introduced new features like the useMultipleSelection hook, which makes it easy to create components that allow users to select multiple items from a list.
>
> Version 6.0.0: This version, released in July 2021, introduced several improvements to the library, including a new useCombobox hook that simplifies the creation of autocomplete components. It also improved the accessibility of the library by providing better support for screen readers and keyboard navigation.
>
> Version 7.0.0: This version, released in March 2022, introduced a new useComboBox hook that provides even more flexibility and customization options for creating autocomplete components. It also added new features like the inputValue prop, which allows developers to control the input value of the component, and improvements to the TypeScript support of the library.